rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/41_knn_search_half_byte_quantized.yml

```diff
@@ -468,7 +468,7 @@ setup:
           query:
             knn:
               field: embedding
-              query_vector: [1.0, 1.0, 1.0, 1.0]
+              query_vector: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
```

Comment:
Would it make sense to use some sort of routing key instead, or would that not properly test the different setups?

Reply:
@pmpailis I would rather not add routing.